LICENSE

```diff
@@ -199,3 +199,387 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
```

Comment:
This license is for the source bundle, which is quite different from the binary bundle.
You can find more information here https://www.apache.org/dev/licensing-howto.html#bundled-vs-non-bundled.

Reply:
I have changed all the issues you mentioned.